ts/Data/Converters/DataConverter.ts

```diff
@@ -69,81 +68,33 @@ class DataConverter implements DataEvent.Emitter {
         switchRowsAndColumns: false
     };
 
-    /* *
-     *
-     *  Static Functions
-     *
-     * */
-
-    /**
-     * Converts an array of columns to a table instance. Second dimension of the
-     * array are the row cells.
-     *
-     * @param {Array<DataTable.Column>} [columns]
-     * Array to convert.
-     *
-     * @param {Array<string>} [headers]
-     * Column names to use.
-     *
-     * @return {DataTable}
-     * Table instance from the arrays.
-     */
-    public static getTableFromColumns(
-        columns: Array<DataTable.Column> = [],
-        headers: Array<string> = []
-    ): DataTable {
-        const table = new DataTable();
-
-        for (
-            let i = 0,
-                iEnd = Math.max(headers.length, columns.length);
-            i < iEnd;
-            ++i
-        ) {
-            table.setColumn(
-                headers[i] || `${i}`,
-                columns[i]
-            );
-        }
-
-        return table;
-    }
-
     /* *
      *
      *  Constructor
      *
      * */
 
     /**
-     * Constructs an instance of the Data Converter.
-     *
-     * @param {DataConverter.Options} [options]
-     * Options for the Data Converter.
+     * Constructs an instance of the DataConverter.
      *
-     * @param {DataConverter.ParseDateCallbackFunction} [parseDate]
-     * A function to parse string representations of dates
-     * into JavaScript timestamps.
+     * @param {DataConverter.UserOptions} [options]
+     * Options for the DataConverter.
      */
     public constructor(
-        options?: Partial<DataConverter.Options>,
-        parseDate?: DataConverter.ParseDateFunction
+        options?: DataConverter.UserOptions
     ) {
-        let decimalPoint;
+        const mergedOptions = merge(DataConverter.defaultOptions, options);
 
-        this.options = merge(DataConverter.defaultOptions, options);
-        this.parseDateFn = parseDate;
+        let regExpPoint = mergedOptions.decimalPoint;
 
-        decimalPoint = this.options.decimalPoint;
+        if (regExpPoint === '.' || regExpPoint === ',') {
+            regExpPoint = regExpPoint === '.' ? '\\.' : ',';
 
-        if (decimalPoint !== '.' && decimalPoint !== ',') {
-            decimalPoint = void 0;
+            this.decimalRegExp =
+                new RegExp('^(-?[0-9]+)' + regExpPoint + '([0-9]+)$', 'u');
```

Comment:
Recently we had a problem with building minified files because of the `u` flag in the regexp, so please remove it :)